iml-gui/configs/webpack.config.js

```diff
@@ -37,7 +37,7 @@ module.exports = (env, argv) => {
       proxy: [
         {
           context: ["/api", "/help"],
-          target: "https://localhost:7444/",
+          target: "https://localhost:8443/",
```

Comment:
I know it's not part of this patch, but we could extract the port to a `const` and use it both here and as a prop to `HtmlWebpackPlugin` so we wouldn't have to update two places.